### PR TITLE
AF-2692 :Add a security permission to show project toolbar, Metrics tab and change request tab

### PR DESCRIPTION
**Pull request by RishiRajAnand**
**Thank you for submitting this pull request**

**AF-2690**: [AF-2690](https://issues.redhat.com/browse/AF-2690)

**Referenced Pull Requests**:
Merge with : 
https://github.com/kiegroup/appformer/pull/1081
https://github.com/kiegroup/kie-wb-distributions/pull/1076

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java [hunk 1]
```diff
@@ -39,11 +39,20 @@
                                                    IsElement {
 
     private static final String DISABLED_CLASS = "disabled";
+    private static final String HIDDEN_CLASS = "hidden";
```
Comment (tomasdavidorg) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java:42]: Not used anywhere.

**Review by tomasdavidorg — CHANGES_REQUESTED** [review]

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java (hunk 1 above)
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java:42]: oh, i might have missed it. Thanks

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java [hunk 2]
```diff
@@ -142,6 +150,15 @@ public void setViewDeploymentDetailsEnabled(boolean enabled) {
         }
     }
 
+    @Override
+    public void viewProjectToolbar(boolean visible) {
+        if (!visible) {
+            this.runTests.hidden = true;
+            this.deployBtn.hidden = true;
+            this.buildBtn.hidden = true;
+        }
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java:159]:
```suggestion
        this.runTests.hidden = !visible;
        this.deployBtn.hidden = !visible;
        this.buildBtn.hidden = !visible;
```

**Review by paulovmr — CHANGES_REQUESTED** [review]

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java (hunk 2 above)
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/project/actions/ProjectMainActionsViewImpl.java]: done

**Review by tomasdavidorg — APPROVED** [review]

**Review by paulovmr — APPROVED** [review]

**Pull request merged**